task2/ConsoleHost.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace task2
+{
+   
+    public static class ConsoleHost
+    {
+        public static int left = Console.WindowWidth / 2 - 5;
+        public static int top  = Console.WindowHeight / 2 - 5;
+        public static void Run()
+        {
+          
+            Console.WriteLine("Hello, What shape do you want to Draw");
+            var input = Console.ReadLine();
+            if (Validator.validateFirstInput(input))
+            {
+                
+                Console.WriteLine("What Color do you like to use for shape ?");
+                Console.WriteLine("You can choose one of these");
+                Console.WriteLine(" White, Blue, Red, Green, Azure, Gray, Gold, Lime, Olive, Orange, Purple, Silver");
```

Comment:
Write method that will generate this message, which will depend on available colors